src/test/java/jsUtils.js

```diff
@@ -23,6 +23,10 @@ function getTomorrowUtcDate(){
     return new Date(new java.util.Date().getTime() + (1000 * 60 * 60 * 24)).getTime();
 }
 
+function getEpochDate(days){
```

Comment:
It would be good if you can add a small documentation of util functions. It can contains 
```
The purpose of the function
@required argument
@ value to be returned
```

Reply:
will do this

Reply:
sounds good

Reply:
can you check now, if this is fine